tensorflow_datasets/image/celebahq.py

```diff
@@ -116,19 +116,20 @@ def _info(self):
 
   def _split_generators(self, dl_manager):
     """Returns SplitGenerators."""
-    image_tar_file = os.path.join(dl_manager.manual_dir,
-                                  self.builder_config.file_name)
-    if not tf.io.gfile.exists(image_tar_file):
-      # The current celebahq generation code depends on a concrete version of
-      # pillow library and cannot be easily ported into tfds.
```

Comment:
Closing this because of a lack of solutions for now.
We can integrate it if the upstream dataset gets updated